cluster/juju/layers/kubernetes-worker/actions/registry

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+# Example usage :
+#
+# Registry creation
+#
+# juju run-action kubernetes-worker/0 registry domain=myregistry.internal \
+# htpasswd="$(cat htpass)" ingress=true \
```

Comment:
The secret used by the registry image needs to be base64 encoded. We can either do it here, or in the code of the action. This is the same for the certificates by the way. What do you think ?